actionview/test/template/test_case_test.rb

```diff
@@ -56,6 +56,10 @@ class GeneralViewTest < ActionView::TestCase
       assert_same _view, view
     end
 
+    test "returns controller_name" do
+      assert_equal controller_name, "test"
```

Comment:
```suggestion
      assert_equal "test", controller_name
```

Reply:
Thanks! I still make this mistake after all these years. 🤦